### PR TITLE
Update Polish (pl) localization of Firefox Hello

### PR DESCRIPTION
**Pull request by piotrdrag**
cc: @splewako

### CODE REVIEW

**Comment by splewako** [comment]
Looks good

File: l10n/pl/add-on.properties [hunk 1]
```diff
@@ -69,50 +68,28 @@ settings_menu_item_turnnotificationsoff=Wyłącz powiadomienia
...
-connection_error_see_console_notification=Nieudane połączenie (szczegóły można znaleźć w konsoli).
 no_media_failure_message=Nie znaleziono kamery ani mikrofonu.
 ice_failure_message=Nieudane połączenie. Zapora sieciowa może blokować połączenia.
 
 legal_text_and_links3=Używając {{clientShortname}}, zgadzasz się na {{terms_of_use}} oraz {{privacy_notice}}.
 legal_text_tos=warunki korzystania z usługi
 legal_text_privacy=politykę prywatności
+
+loop_deprecate_notice_and_link=Firefox Hello niedługo zostanie usunięty z Firefoksa. {{learn_more}}
+loop_deprecate_learn_more=Więcej informacji
```
Comment (piotrdrag) [inline l10n/pl/add-on.properties:81]: @splewako: please review these two new strings.

The original strings: https://github.com/mozilla/loop-client-l10n/blob/master/l10n/templates/add-on.properties#L132

Comment (splewako) [inline l10n/pl/add-on.properties:81]: +1

**Comment by piotrdrag** [comment]
Could you please merge it before the deadline?

**Comment by splewako** [comment]
I can't but @Standard8 should be able to push this.

**Comment by Standard8** [comment]
Thanks, I'd have done it earlier, but I've been afk.

**Pull request merged**